platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/FallenBehindManagerImpl.java

```diff
@@ -46,8 +46,12 @@ public class FallenBehindManagerImpl implements FallenBehindManager, EventCreati
      * fallen behind. This Set is made from a ConcurrentHashMap, so it needs no synchronization
      */
     private final Set<NodeId> notYetReportFallenBehind;
-    /** Called on any fallen behind status change */
-    private final Runnable notifyPlatform;
+
+    /**
+     * A runnable which announces to the platform that this node has fallen behind
+     */
+    private final Runnable announceFallenBehind;
```

Comment:
I wonder if it would be better to pass in a `PlatformStatusStateMachaine`, and just let this class send the appropriate action.

Reply:
Done